packages/compile-solidity/compilerSupplier/loadingStrategies/VersionRange.js

```diff
@@ -4,14 +4,20 @@ const originalRequire = require("original-require");
 const axios = require("axios").default;
 const semver = require("semver");
 const solcWrap = require("solc/wrapper");
-const LoadingStrategy = require("./LoadingStrategy");
 const Cache = require("../Cache");
 const observeListeners = require("../observeListeners");
 const { NoVersionError, NoRequestError } = require("../errors");
 
-class VersionRange extends LoadingStrategy {
-  constructor(...args) {
-    super(...args);
+class VersionRange {
+  constructor(options) {
+    const defaultConfig = {
+      compilerRoots: [
+        "https://relay.trufflesuite.com/solc/bin/",
```

Comment:
Oh, yeah, I didn't notice that...

Reply:
It redirects (302 Found) to the [now outdated] Solidity URL for these. It's there in case solc's own distribution methods fail; we can switch to hosting the compilers ourselves at this URL.

Reply:
That's important context! Can you add it as a comment please?